### ISSUE TITLE
client.lives.uploadThumbnail hangs with no result

### ISSUE THREAD

**Issue by beurnii** [apivideo/nodejs-sdk#47]
I am trying to use the uploadThumbnail function for livestream, but it does not work,
My code is super simple: 

> async function thumbnail() {
	const client = new apiVideo.Client({ apiKey: 'key' });
	client.lives.uploadThumbnail('thumbnail.jpeg', 'stream id')
}

The process seems to hang on this [line](https://github.com/apivideo/nodejs-sdk/blob/14b431155d11b356dbc980d92089045065591e8e/lib/Browser/browser.js#L119).

If anyone has a solution, please let me know. Thanks!

**Comment by lutangar** [comment]
Hello @beurnii ,

I added a new test case in #48 and it seems to work as expected:
```js
const live = await client.lives.uploadThumbnail('test/data/test.jpg', liveStreamId);
```

What' your `nodejs` version?

Do you actually have an error or does it hang forever?
If you experience this, it might be only a usage/syntax issue. Maybe you simply forget to add a `return` statement?
```js
async function thumbnail() {
  const client = new apiVideo.Client({ apiKey: 'key' });
  const live = await client.lives.uploadThumbnail('thumbnail.jpeg', 'stream id');
  console.log(live);
  return live;
}
```

**Comment by beurnii** [comment]
Hi @lutangar 

My node is 15.10.0
My api.video api is 1.10.2
My got is 11.8.2

There is no error message, it does really hang forever. I let it run for an hour without any result. 
If I put it in a try catch, there is no result either, it simply hangs.
I had a co-worker test it and he had the some problem.

Thanks

**Comment by lutangar** [comment]
Test is passing on node 15 as well.

Did you adapt your code to the suggestion in previous comment?
If so, could you provide the revised version of your code?

**Comment by beurnii** [comment]
Here my code 
```node
async function thumbnail() {
	console.log('start')
	const client = new apiVideo.ClientSandbox({ apiKey: 'awKvKphj2yPgzOQ6NeVh564mqHCnvZyy4P4KxwHJNAf' });
	const live = await client.lives.uploadThumbnail('thumbnail.jpg', 'li2fnO5UQJpsoGwDM9r4NeIy');
	console.log(live);
	console.log('end')
}
```
With this code, the console log with live and end are never run.

I tried running the api test on my device and they all pass. I am not sure what could be the problem.

**Comment by beurnii** [comment]
@lutangar thanks for the edit

**Comment by lutangar** [comment]
Alright, the good news is I managed to reproduce your error.

But I don't know quite yet what's happening, but I noticed the following does work:
```js
const apiVideo = require('@api.video/nodejs-sdk');

async function thumbnail() {
  console.log('start')
  const client = new apiVideo.ClientSandbox({ apiKey: 'xxx' });
  const { liveStreamId } = await client.lives.create('This is a live');
  const live = await client.lives.uploadThumbnail('test/data/test.jpg', liveStreamId);
  console.log(live);
  console.log('end');
}


(async () => {
  try {
    await thumbnail();
  } catch (e) {
    console.error(e);
  }
})();
```

It seems having another call, here `client.lives.create`, make the subsequent call, here `client.lives.uploadThumbnail`, works :thinking:

Thanks for reporting, I'll let you know when it's fixed.

**Comment by beurnii** [comment]
Looks like it works when another function primed the auth token. 
I will use the client.lives.get(id) before uploading my thumbnail. 

Thanks for the help!

**Comment by lutangar** [comment]
Should be fixed in 1.10.3, let us know if you're still experiencing the issue.
